.gitlab-ci.yml

```diff
@@ -33,6 +33,11 @@ run-tests:
     # start the docker daemon and give it some time to start:
     - dockerd-entrypoint.sh > /dev/null 2>&1 & sleep 2
 
+  artifacts:
+    paths:
+      - logs
+    when: always
```

Comment:
Maybe `on_failure` would be more appropriate?

Reply:
It doesn't hurt to keep them, they could be used as a reference for someone wanting to check what the logs look like in a normal flow. They are relatively small and will be deleted after some time anyway.
What do you think?